app/scripts/modules/core/src/domain/ILoadBalancer.ts

```diff
@@ -31,3 +38,12 @@ export interface ILoadBalancerGroup {
   subgroups?: ILoadBalancerGroup[];
   searchField?: string;
 }
+
+export interface IUpsertLoadBalancerCommand {
+  credentials: string;
+  detail?: string;
+  editMode?: boolean; // Is this even used anymore?
```

Comment:
one way to find out...
fwiw, full text search across orca and clouddriver, doesn't show up in clouddriver, and only shows up in a test in orca.